### ISSUE TITLE
killed by SIGTRAP immediately after launch

### ISSUE THREAD

**Issue by AGSPhoenix** [timeseriesadmin/timeseriesadmin#479]
```
phoenix@host:~$ timeseriesadmin 
Trace/breakpoint trap
phoenix@host:~$ uname -a
Linux sabre 4.19.0-11-amd64 #1 SMP Debian 4.19.146-1 (2020-09-17) x86_64 GNU/Linux
phoenix@host:~$ cat /etc/debian_version 
10.6
phoenix@host:~$ strace timeseriesadmin
[...]
clone(child_stack=NULL, flags=CLONE_CHILD_CLEARTID|CLONE_CHILD_SETTID|SIGCHLD, child_tidptr=0x7ffbbdc5e010) = 15118
rt_sigprocmask(SIG_SETMASK, [], ~[KILL STOP RTMIN RT_1], 8) = 0
close(11)                               = 0
close(10)                               = 0
recvmsg(9, {msg_name=NULL, msg_namelen=0, msg_iov=[{iov_base="", iov_len=13}], msg_iovlen=1, msg_controllen=0, msg_flags=0}, 0) = 0
--- SIGTRAP {si_signo=SIGTRAP, si_code=SI_KERNEL} ---
+++ killed by SIGTRAP +++
Trace/breakpoint trap
phoenix@host:~$ 
```
Version is 0.1.13.

**Comment by jmarceli** [comment]
Hi @AGSPhoenix,
The timeseriesadmin is not meant to be run from a command line. You can use either Docker to run it or install it as an Electron application (see https://timeseriesadmin.github.io/#download).

**Comment by AGSPhoenix** [comment]
Yes, I installed it from that link.

Tried launching it from the GUI to start with, and it never showed up. The CLI output shows what seems to be the issue.

I'll draw your attention to the fact that I'm running Debian, if you didn't see it. It seemed to install without any issues, but there may be weird package version issues behind this.

**Comment by jmarceli** [comment]
@AGSPhoenix  thanks for reporting that. Indeed there is some bug with a .deb package I will do my best to fix it soon. If you need a working copy right now please use Docker based version or a legacy on e.g. https://github.com/timeseriesadmin/timeseriesadmin/releases/download/v0.1.8/TimeSeriesAdmin.deb

**Comment by jmarceli** [comment]
@AGSPhoenix I have tried once more to reproduce your issue on Ubuntu but with no luck this time. I was able to install newest TimeSeriesAdmin using `sudo dpkg -i TimeSeriesAdmin.deb`, then I was able to remove it properly and install even via GUI. I was testing this using Ubuntu 18.04, maybe Debian is not supporting Electron apps so well as Ubuntu does.
I'm sorry but I think I'm not able to help you with this.
I thought it is a general .deb package issue but it's not.
The only thing I can suggest is running through a Docker container, which I understand might be not the most convenient way, but definitely most reliable one.
